### PR TITLE
webapp/partials/*.jsp: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352

### CODE REVIEW

File: src/main/webapp/partials/instructorHelpComments.jsp [hunk 1]
```diff
@@ -1,209 +1,209 @@
...
+                              <tbody>
+                                <tr>
+                                  <th class="text-center">User/Group</th>
+                                  <th class="text-center">Can see this comment</th>
+                                  <th class="text-center">Can see comment giver's name</th>
+                                </tr>
+                                <tr id="response-giver-8-0-1">
+                                  <td class="text-left">
+                                    <div data-toggle="tooltip" data-placement="top" title="Control what response giver can view">
+                                      Response Giver</div>
```
Comment (chowyb) [inline src/main/webapp/partials/instructorHelpComments.jsp]: Move `</div>` to the next line.

File: src/main/webapp/partials/instructorHelpComments.jsp [hunk 2]
```diff
@@ -1,209 +1,209 @@
...
-                                                                        <input class="visibilityCheckbox answerCheckbox" type="checkbox" value="INSTRUCTORS" checked="">
-                                                                    </td>
-                                                                    <td>
-                                                                        <input class="visibilityCheckbox giverCheckbox" type="checkbox" value="INSTRUCTORS" checked="">
-                                                                    </td>
-                                                                </tr>
+                                <tr id="response-recipient-8-0-1">
+                                  <td class="text-left">
+                                    <div data-toggle="tooltip" data-placement="top" title="Control what response recipient(s) can view">
+                                      Response Recipient(s)</div>
```
Comment (chowyb) [inline src/main/webapp/partials/instructorHelpComments.jsp]: Move `</div>` to the next line.

File: src/main/webapp/partials/instructorHelpComments.jsp [hunk 3]
```diff
@@ -1,209 +1,209 @@
...
-                                                </div>
-                                            </form>
-                                        </li>
-                                    </ul>
-                                </div>
-                            </div>
+                                <tr id="response-instructors-8-0-1">
+                                  <td class="text-left">
+                                    <div data-toggle="tooltip" data-placement="top" title="Control what instructors can view">
+                                      Instructors</div>
```
Comment (chowyb) [inline src/main/webapp/partials/instructorHelpComments.jsp]: Move `</div>` to the next line.

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/partials/instructorHelpFAQ.jsp [hunk 4]
```diff
@@ -1,198 +1,204 @@
...
+          <a name="faq11" id="question">
+            How are the contribution numbers calculated?
+          </a>
+        </b>
+        <div class="helpSectionContent">
+          You can find the details of the contribution calculation scheme
+          <a href="https://docs.google.com/document/d/1hjQQHYM3YId0EUSrGnJWG5AeFpDD_G7xg_d--7jg3vU/pub?embedded=true#h.n5u2xs6z9y0g">here</a>.
+        </div>
+      </li>
+        <br>
```
Comment (chowyb) [inline src/main/webapp/partials/instructorHelpFAQ.jsp]: Missed this: remove 2 spaces from this line.

Comment (sujeet14108) [reply src/main/webapp/partials/instructorHelpFAQ.jsp]: Done

**Pull request merged**